test/test_error_responses.py

```diff
@@ -0,0 +1,200 @@
+"""
```

Comment:
I wouldn't have all the error tests in a single file. I think they should be put into their corresponding test files; e.g. tests testing dumping errors should go to the test_dump.py file. Tell me if you don't think some tests don't have a corresponding test file.

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/81/commits/c83f3afe5b84a5da50fd55c6cd98b9fa28006090